scalafix/input/src/main/scala/fix/LinearSeqSrc.scala

```diff
@@ -1,10 +0,0 @@
-/*
-rule = "scala:fix.CrossCompat"
- */
-package fix
-
-import scala.collection.LinearSeq
-
-object LinearSeqSrc {
-  LinearSeq(1)
-}
```

Comment:
We should still support this case

Reply:
https://github.com/scala/scala-collection-compat/issues/99#issue-341891540

Reply:
No, if it's incorrect we should remove it. I'm using it to migrate akka, if we keep this rewrite in It's not possible to use scala-collection-compat to rewrite code.